src/Url.php

```diff
@@ -61,7 +61,13 @@ public function unsignedPath(): string
         return "/{$this->fit}/{$this->w}/{$this->h}/{$this->gravity}/{$enlarge}/{$encodedUrl}" . ($ext ? ".$ext" : "");
     }
 
-    public function signedPath(): string
+    public function insecureSignedPath(): string
+    {
+        $unsignedPath = $this->unsignedPath();
+        return "/insecure$unsignedPath";
+    }
+
+    public function secureSignedPath(): string
     {
         $unsignedPath = $this->unsignedPath();
```

Comment:
This line should be moved to "signedPath()". Both `secureSignedPath()` and `insecureSignedPath()` should get a parameter `$sourcePath` and use that. Reduces code duplication